### PR TITLE
remove global.NamespacePrefix

### PR DESCRIPTION
**Pull request by philbrookes**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -177,42 +177,66 @@ image/build/test:
...
 
 .PHONY: test/e2e/local
+test/e2e/local: export WATCH_NAMESPACE := $(NAMESPACE)
 test/e2e/local: cluster/cleanup cluster/cleanup/crds cluster/prepare cluster/prepare/crd deploy/integreatly-rhmi-cr.yml
 	$(OPERATOR_SDK) --verbose test local ./test/e2e --watch-namespace="$(NAMESPACE)" --operator-namespace="${NAMESPACE}" --go-test-flags "-timeout=120m" --debug --up-local
 
+.PHONY: test/e2e/regex
+test/e2e/regex: export WATCH_NAMESPACE := $(NAMESPACE)
+test/e2e/regex:
+	go clean -testcache && go test -v ./test/functional -run="$REGEX" -timeout=80m
```
Comment (CathalOConnorRH) [inline Makefile:223]: I think you are missing brackets around regex
	go clean -testcache && go test -v ./test/functional -run="$(REGEX)" -timeout=80m

**Comment by philbrookes** [comment]
/override ci/prow/rhoam-e2e
Only one flaky test failed, that passed locally for me.

**Pull request merged**